src/index.js

```diff
@@ -120,10 +120,9 @@ export default class Spinner extends React.Component {
   _renderSpinner() {
     const { visible } = this.state;
 
-    if (!visible)
-      return (
-        <View />
-      );
+    if (!visible) {
+      return null;
```

Comment:
can you just make this `if (!visible) return`?  `return` by itself returns `null`